Dockerfile

```diff
@@ -32,12 +34,13 @@ RUN apt-get update -y && \
       libtool \
       make \
       openssl \
-      python3.8 \
-      python3.8-dev \
+      python3.12 \
+      python3.12-dev \
       python3-pip \
       unzip \
       vim-tiny && \
-    pip install --no-cache-dir --upgrade pip setuptools && \
+    pip install --no-cache-dir "modules/reana-commons[yadage]" --upgrade && \
```

Comment:
I don't think we need to manually install reana-commons here, do we? Let's keep this command as it was before, it should work fine when a new release of reana-commons upgrading the yadage dependencies will be published on PyPI